core/spec.md

```diff
@@ -333,6 +366,13 @@ Therefore, the hierarchical structure of the Registry model is defined in such
 a way that it can be represented in a single file, including but not limited
 to JSON, or via the entity graph of a REST API.
 
+If an error occurs during the processing of a request, even if the error was
+during the creation of the response (e.g. an invalid `?inline` value was
+provided), then an error MUST be generated and the entire request (even any
+update actions) MUST be undone. In general, when an error is generated it
+SHOULD be sent back to the client. However, this MAY not happen if the server
```

Comment:
Presumably if an error occurs, the server still has to send *an* error back to the client, but it doesn't need to have full information?

Reply:
I think we need to leave it as an impl choice since there are some systems that might think it's best to just close the connection rather than even give any error message at all - again for security reasons. I think this could be one of those "self correcting issues" where if people make poor choices then people will stop using that system. This is why I prefer to use the phrase "generate an error" rather than "send (or respond with) an error"... keeps the spec flexible.